src/main/java/com/bean00/Driver.java

```diff
@@ -17,25 +17,27 @@
 public class Driver {
 
     public static void main(String[] args) throws IOException {
+        DataStore dataStore = new FileSystemDataStore("");
+        int portNumber;
         String pathToRoot;
 
         try {
-            ArgParser argParser = new ArgParser();
+            ArgParser argParser = new ArgParser(dataStore);
             ServerOptions options = argParser.buildServerOptions(args);
+            portNumber = options.getPort();
             pathToRoot = options.getDirectory();
         } catch (IllegalArgumentException e) {
             printErrorMessage();
```

Comment:
What's the point of catching this exception if you are just doing the same thing it would do if you didn't catch anything?

Reply:
So the reason I'm also catching the exception here is that I wanted to force the program to stop running if an `IllegalArgumentException` is thrown. So if I don't catch anything, the program continues with the `println` statements. But if I do catch something, it prints out the error message and exits gracefully.

Reply:
Oops. When I read that line I thought it was `e.printStackTrace()`.